README.md

```diff
@@ -1,64 +1,92 @@
-# **OSSF Endusers**
+# OSSF End Users
 
-[Brief description of the initiative]
+Anyone is welcome to join our open discussions related to the group's mission and charter.
 
+## Quick Start
 
-## 
-**Motivation**
+### Meeting times
 
-[Background / use cases of the problem to be solved]
+Every 2 weeks, Thursday 10am EST/3pm UTC. The meeting invite with zoom details is available on the [public OSSF calendar](https://calendar.google.com/calendar?cid=czYzdm9lZmhwNWk5cGZsdGI1cTY3bmdwZXNAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ)
 
+### Meeting notes
 
-## 
-**Objective**
+Meeting notes are [maintained in a Google Doc](https://docs.google.com/document/d/1KQalBRzfRBvsqh73JUYfp1KG-AJdXcv2Z8LTIFoQP8c/edit#). If attending please add your name, and if a returning attendee, please change the color of your name from gray to black.
 
-[What is to be achieved with this initiative]
+### Areas that need contributions
 
-[OKRs - OPTIONAL]
+- WG Documentation
+- Others TBD
 
+### Where to file issues
 
-## 
-**Scope**
+Issues can be reviewed and filed [here](https://github.com/ossf/wg-endusers/issues)
 
-[What is in and out of scope]
+## Objective
 
+The End User Working Group aims to ensure that the distinct and impactful voice of end users is heard in the development and delivery of the technical vision of OpenSSF.
 
-## 
-**Prior Work**
+## Vision
 
+The End Users Working Group (WG) represents the interests of public and private sector organizations that primarily consume open source rather than produce it.
 
+We strive to:
 
-*   List of prior and/or related projects
+- Ensure that the use cases for end user consumption of open- source software are understood and factored into OSSF programs.
+- Provide the resources required by End Users to develop and implement more efficient and effective strategies, processes, tools, best practices and solutions that secure software supply-chains
+- Provide a forum for learning from the experience and insights of peers
+- Create an End Users working group, with representation from key private industry, public sectors, and multiple geographical regions.
+- Establish user representation and active participation in OpenSSF working groups and leadership, both in the TAC and the Governing Board
```

Comment:
Nit: this item does not end with a period.

Reply:
Fixed.